### DISCUSSION TITLE
Serializable DKG secret packages

### DISCUSSION THREAD

**Discussion by 0x330a** [ZcashFoundation/frost#711] [Ideas]
It would be great to also allow serialization of the round 1 / 2 secret packages so that the key generation didn't have to be all at once in the same process for more asynchronous key generation. Currently trying to figure out how to do this as we can get the coefficients in round one but not reconstruct a SecretPackage outside of the library. Maybe I am missing something but I would be interested if anyone else has encountered or overcome this

**Comment by conradoplg** [comment]
Sorry this fell through the cracks. Our reasoning was simply "let's not provide this for now to make it harder for people to persist secrets when not needed" but it seems a common scenario we need to support. I've created an issue for it and added to our backlog https://github.com/ZcashFoundation/frost/issues/733

**Reply by 0x330a** [reply]
Thanks for the issue, I'll subscribe there :+1:
